cmd/skopeo/utils.go

```diff
@@ -9,12 +9,20 @@ import (
 	"github.com/urfave/cli"
 )
 
-func contextFromGlobalOptions(c *cli.Context, credsFlag string) (*types.SystemContext, error) {
+func contextFromGlobalOptions(c *cli.Context, credsFlag, certPathFlag, tlsVerifyFlag string) (*types.SystemContext, error) {
```

Comment:
This is unwieldy… can this instead be a `flagPrefix string` (`""`, "src-"`, "dest-"`)?